src/sea.h

```diff
@@ -1,8 +1,7 @@
-#pragma once
-#include <stdio.h>
+#ifndef _SEA_H
```

Comment:
re: standards, c99? I think we should shoot for greatest portability, keep it simple, plain C. e.g., no `_GNU_SOURCE` extensions.